### PR TITLE
[Core] Hotfix define python for tensor

### PR DESCRIPTION
**Pull request by loumalouomega**
The tensor definition in python is wrong, the module definition is missing. Additionally I removed the python defoinitions from define.h, that are already defined in define_python. Additionally in the define.h the python module is missing

### CODE REVIEW

**Comment by loumalouomega** [comment]
Finally I discover the origin of the error, it was the lack of corresponding KratosComponents for the tensors

**Comment by loumalouomega** [comment]
I added other things missing for proper definition of tensors (basically all the definitions of the array components are missing)

File: kratos/includes/define.h [hunk 1]
```diff
@@ -562,36 +562,6 @@ catch(...) { Block KRATOS_THROW_ERROR(std::runtime_error, "Unknown error", MoreI
...
-#endif
-#define KRATOS_REGISTER_IN_PYTHON_FLAG_IMPLEMENTATION(flag) \
-    scope().attr(#flag) = std::ref(flag)      \
- 
-#ifdef KRATOS_REGISTER_IN_PYTHON_FLAG
-#undef KRATOS_REGISTER_IN_PYTHON_FLAG
-#endif
-#define KRATOS_REGISTER_IN_PYTHON_FLAG(flag) \
-    KRATOS_REGISTER_IN_PYTHON_FLAG_IMPLEMENTATION(flag);   \
-    KRATOS_REGISTER_IN_PYTHON_FLAG_IMPLEMENTATION(NOT_##flag)
```
Comment (roigcarlo) [inline kratos/includes/define.h]: @KratosMultiphysics/technical-committee Can we delete this safely?

Comment (loumalouomega) [reply kratos/includes/define.h]: @roigcarlo I can revert this change and I can do it in a different PR, but this PR is important or if anyone tries to use a tensor variable will have linking problem with pybind

**Comment by loumalouomega** [comment]
Reverted, I will create a different PR for that

**Comment by loumalouomega** [comment]
@roigcarlo can you look at this?

**Review by roigcarlo — APPROVED** [review]

**Pull request merged**